moseq2_model/helpers/wrappers.py

```diff
@@ -170,6 +172,65 @@ def learn_model_wrapper(input_file, dest_file, config_data):
         return img_path
 
 
+def apply_model_wrapper(model_file, pc_file, dest_file, config_data):
+    """
+    Wrapper function to apply a pre-trained model to new data.
+
+    Args:
+    model_file (str): Path to pre-trained model file
+    pc_file (str): Path to PC scores file
+    dest_file (str): Path to save output file
+
+    Returns:
+    None
+    """
+
+    assert splitext(basename(dest_file))[-1] in ['.mat', '.z', '.pkl', '.p', '.h5'], 'Incorrect model filetype'
+    os.makedirs(dirname(dest_file), exist_ok=True)
+
+    if not os.access(dirname(dest_file), os.W_OK):
+        raise IOError('Output directory is not writable.')
+
+
+    # Load model
+    model_data = load_dict(model_file)
+
+    if model_data.get('whitening_parameters') is None:
+        raise KeyError('Whitening parameters not found in model file. Unable to apply model to new data. Please retrain the model using the latest version.')
```

Comment:
`if 'whitening_parameters' not in model_data` is more pythonic.

Reply:
I thought about that but ended up implementing it this way because this also handles `model_data` having the `whitening_paramenters` key but `model_data['whitening_parameters']` is `None`.